lib/dropbox_api/metadata/root_info.rb

```diff
@@ -0,0 +1,16 @@
+module DropboxApi::Metadata
```

Comment:
@Jesus I have followed the format set out in the other files for 
> Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.
Do you want this changed?

Reply:
@Jesus Is there anything I can do to help get this PR merged?